### ISSUE TITLE
0995 - Release authorization alert does not announce as an alert for screen readers

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#109301]
## Issue
Finding from recent mini audit of the 4142 in 0995

On the release authorization page when the alert is triggered, focus moves up to the alert, but it's not announced as an alert. Add `role="alert"` to the design system component to get it to announce for screen readers.

## Resources
https://design.va.gov/components/alert/#assign-an-appropriate-aria-role
